triggers/service/src/test/scala/com/digitalasset/daml/lf/engine/trigger/TriggerServiceTest.scala

```diff
@@ -501,7 +501,10 @@ trait AbstractTriggerServiceTest extends AbstractTriggerServiceTestHelper {
     } yield succeed
   }
 
-  it should "restart trigger on initialization failure due to failed connection" taggedAs availabilitySecurity inClaims withTriggerService(
+  it should "restart trigger on initialization failure due to failed connection" taggedAs availabilitySecurity
+    .setHappyCase(
```

Comment:
I think this is more about reliability than security. Could you please change the tags, so this ends up in the right bucket?

Reply:
Yes, that's fine!

Reply:
As discussed, going to merge without changing to `ReliabilityTest`.